server/container/util/src/main/java/org/apache/james/util/scanner/SpamAssassinResult.java

```diff
@@ -80,20 +80,26 @@ public SpamAssassinResult build() {
                 headersAsAttribute.put(STATUS_MAIL_ATTRIBUTE_NAME, "No, hits=" + hits + " required=" + requiredHits);
             }
 
-            return new SpamAssassinResult(hits, requiredHits, headersAsAttribute.build());
+            return new SpamAssassinResult(isSpam, hits, requiredHits, headersAsAttribute.build());
         }
     }
 
+    private final boolean isSpam;
     private final String hits;
     private final String requiredHits;
     private final Map<String, String> headersAsAttribute;
 
-    private SpamAssassinResult(String hits, String requiredHits, Map<String, String> headersAsAttribute) {
+    private SpamAssassinResult(boolean isSpam, String hits, String requiredHits, Map<String, String> headersAsAttribute) {
```

Comment:
we should consider boolean as a code smell in general and use a type every time

Reply:
Maybe it's time to enhance this class:
 - cast _hits_ & _requiredHits_ to double in **Builder**
 - _isSpam_ is _hits_ >= _requiredHits_

Reply:
This modification has been reverted.